Cargo.toml

```diff
@@ -12,10 +12,16 @@ autobenches = false
 [lib]
 bench = false
 
+[features]
+default = [ "std" ]
```

Comment:
Oh, just in case: With the above strategy, `serde` needs to be configured with `features = ["derive", "alloc"]` - otherwise you'll get strange error messages.

Reply:
> Be more open towards which versions of `hashbrown` are supported. I think `hashbrown = "0.14"` (Cargo.toml) should do the trick? I *think* (really not sure) this will allow Cargo to unify different hashbrown versions more easily (reducing compilation times).

Just a comment from the sidelines: I think it's best practice to specify the full version, as this PR is already doing.

Short explanation:

- With a less strict version, you increase the risk of Cargo resolving to a version that you don't support (because you're using features from a newer one).
- If the version is specified without an `=` in front, it means "this version *or higher* (while respecting semver rules)" anyway, so it's not like you're pinning to a specific version.

Longer explanation: https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277

Reply:
Ah, that's good to know. I appreciate the remark!
Let's leave the version at `hashbrown = "0.14.2"`.

Reply:
Made changes. Please have a look when you have time, I can make more changes if necessary

Reply:
Sorry I just saw your comment about `serde` not `spade`. I honestly overlooked that `serde` requires configuration for no_std -.- Changed that too

Reply:
All looks good! Thanks again for your contribution and the discussions. I'll merge soon and publish a new version.